src/galaxy/validation/models.py

```diff
@@ -599,3 +599,18 @@ class UserStatistics(BaseModel):
     modified_highway : int  
     added_highway_meters : float
     modified_highway_meters : float
+
+class DataSource(str, Enum):
+    UNDERPASS = "underpass"
+    INSIGHTS = "insight"
+
+class DataOutput(str, Enum):
+    OSM = "osm"
```

Comment:
if possible it would be good if we can maintain same variable naming , OSM to osm , by looking at another variable names !

Reply:
Is it better to use planet in this instance or some other variable? cc: @kshitijrajsharma